packages/core/src/cell/cell.scss

```
@@ -75,10 +81,17 @@
+  &--active {
+    background: $cell-active-color;
+  }
+
   &__value {
-    flex: 1;
+    position: relative;
+    overflow: hidden;
+    font-size: $cell-value-font-size;
     color: $cell-value-color;
     text-align: right;
+    word-wrap: break-word;
```

Stylelint (17.14.0)

[error] Expected "word-wrap" to be "overflow-wrap" (property-no-deprecated)

(property-no-deprecated)